source/apiVolontaria/apiVolontaria/urls.py

```diff
@@ -41,6 +41,14 @@
         UsersId.as_view(),
         name='users_id',
     ),
+    url(
+        r'^profile$',
+        UsersId.as_view(),
+        kwargs=dict(
+            profile=True,
+        ),
```

Comment:
Does this option mean you always send an extra argument `profile` set to `True` ? Without it the profile is not returned?

Reply:
Yeah, it add a profile set to true all the time. The goal is to use the `UsersId` view with this special kwargs to change the default behaviour.